cdap-api/src/main/java/co/cask/cdap/api/lineage/operation/Input.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.lineage.operation;
+
+/**
+ * Represent the input with name and the origin(operation) which created this input.
+ */
+public class Input {
```

Comment:
Can we call it InputField? That way it won't clash names with the existing Input class, and also makes it clear that this is about a field, not a dataset.

Reply:
calling InputField.